src/main/java/org/embulk/util/config/ConfigMapperFactory.java

```diff
@@ -320,6 +323,39 @@ private ObjectMapper mapperForOthers() {
         return objectMapper;
     }
 
+    private static void assertJacksonCoreVersion() {
+        if (com.fasterxml.jackson.core.json.PackageVersion.VERSION.getMajorVersion() != 2) {
+            throw new UnsupportedOperationException("embulk-util-config is not used with Jackson 2.");
+        }
+
+        final int minor = com.fasterxml.jackson.core.json.PackageVersion.VERSION.getMinorVersion();
+        if (minor < 14 || (minor == 15 && com.fasterxml.jackson.core.json.PackageVersion.VERSION.getPatchLevel() <= 2)) {
+            throw new UnsupportedOperationException("embulk-util-config is not used with Jackson (jackson-core) 2.15.3 or later.");
+        }
+    }
+
+    private static void assertJacksonDataBindVersion() {
+        if (com.fasterxml.jackson.databind.cfg.PackageVersion.VERSION.getMajorVersion() != 2) {
+            throw new UnsupportedOperationException("embulk-util-config is not used with Jackson 2.");
```

Comment:
I've missed point out this part. 

Please create another PR when this change useful.
`embulk-util-config is not used with Jackson(jackson-databind) 2.`

Reply:
I basically don't think it's needed for this part.

Jackson 1 had different Java package names (`org.codehaus.jackson.*`) and Jackson 3 is going to have yet another Java package names (`tools.jackson.*`). While using Jackson 2 (`com.fasterxml.jackson.*`), the major version should be always 2.

This part is practically just an assertion. When it happens, that's not a simple problem anyway. ;)

Reply:
I got it. Thanks! LGTM👍